js/src/dapps/tokenreg/Inputs/Text/input-text.js

```diff
@@ -105,22 +105,22 @@ export default class InputText extends Component {
     const { validationType, contract } = this.props;
     const validation = validate(value, validationType, contract);
 
-    if (validation instanceof Promise) {
```

Comment:
@gavofyork In JS, it is considered bad practice to assert `instanceof` in these cases. We only care about getting a thenable interface, which `Promise.resolve` should happily accept. See https://github.com/then/is-promise/issues/6